.github/workflows/test-integrations.yml

```diff
@@ -549,7 +549,7 @@ jobs:
             -json ./... \
             --target-image ${{ env.CONSUL_LATEST_IMAGE_NAME }} \
             --target-version local \
-            --latest-image docker.mirror.hashicorp.services/${{ env.CONSUL_LATEST_IMAGE_NAME }} \
+            --latest-image hashicorp/${{ env.CONSUL_LATEST_IMAGE_NAME }} \
```

Comment:
@loshz , is it ok to switch from `docker.mirror` to `hashicorp`?

Reply:
https://github.com/docker-library/docs/pull/2283 is where we stop publishing to regular docker hub. If the mirror is truly just a mirror, it would make sense that it also doesn't have the tag. 

But I suppose we could switch to `docker.mirror.hashicorp.services/hashicorp/consul`! Let me try that